### ISSUE TITLE
Process will disappear in taskmgr

### ISSUE THREAD

**Issue by Simba98** [sandboxie-plus/Sandboxie#3043]
### Describe what you noticed and did

I run Fritz 17 and load stockfish 15.1 in sandboxie
And I can not find the stockfish process in the taskmgr, while I can find the process in resource monitor.
![image](https://github.com/sandboxie-plus/Sandboxie/assets/7941967/2b0c47b0-0f19-4573-b9b1-5f44e1c53152)
![image](https://github.com/sandboxie-plus/Sandboxie/assets/7941967/332cc61c-a73f-4d6b-9c31-cd265b5d5621)

Actually, my stockfish is stuck at the E-Cores in 12th Core Arch, so what I wanted is change it to P-Cores.
But I can not due to it is missing.

I try to find out a better way to reproduce it, but I did not find any related way to do so, e.g. run stockfish from cmd.exe.

### How often did you encounter it so far?

Everytime I launch stockfish

### Affected program

Fritz 17 and stockfish 15.1

### Download link

https://stockfishchess.org/download/

### Where is the program located?

The program is installed only inside a sandbox (NOT in the real system anyway).

### Expected behavior

Appear in taskmgr (and run in P-core, but E/P-core may not be related to sandboxie)

### What is your Windows edition and version?

Windows 11 22621.1702 Home

### In which Windows account you have this problem?

A Microsoft account (Administrator).

### Please mention any installed security software

Microsoft Defender

### What version of Sandboxie are you running?

Can be reproduced from 1.5.2 to 1.9.6

### Is it a new installation of Sandboxie?

I just updated Sandboxie from a previous version (I remember which one it is).

### Is it a regression?

Since 1.5.2

### In which sandbox type you have this problem?

In a standard isolation sandbox (yellow sandbox icon).

### Can you reproduce this problem on a new empty sandbox?

I can confirm it also on a new empty sandbox.

### Did you previously enable some security policy settings outside Sandboxie?

Appear in taskmgr (and run in P-core, but E/P-core may not be related to sandboxie)

**Comment by DavidXanatos** [comment]
that's strange, are you vieving processes of all users in task manager?

**Comment by Simba98** [comment]
> that's strange, are you vieving processes of all users in task manager?

Yes, I was viewing all processes, as you can see the taskmgr showed the svchost.exe and showing svchost needs to be promoted.
The account I used is a local Admin and I reproduced it on Windows 10 19045.3086.

**Comment by Simba98** [comment]
I tried some other methods:

From un-elevated cmd.exe (but in Admin account):
```
tasklist | findstr 10808
stockfish-windows-2022-x8    10808 Console                    1  4,821,032 K
```
Find stockfish.

From un-elevated powershell
```
Get-Process  | Where-Object {$_.Id -eq 10808}

Handles  NPM(K)    PM(K)      WS(K)     CPU(s)     Id  SI ProcessName
-------  ------    -----      -----     ------     --  -- -----------
     60       8  4903488    4829508     404.47  10808   1 stockfish-windows-2022-x86-64-avx2
```
Also found.

**Comment by Simba98** [comment]
I enabled the Sandboxie token Option and find only the chess engine will hide from taskmgr.
So I guess this is due to the way that how fritz start process.
![image](https://github.com/sandboxie-plus/Sandboxie/assets/7941967/83eca628-28b7-48c8-ab56-ea61bf79723a)
![image](https://github.com/sandboxie-plus/Sandboxie/assets/7941967/a5e55545-5ea5-4ba1-ac4f-2576eb4300ba)

And only the stockfish will be hide in taskmgr.
